### ISSUE TITLE
error spam :<

### ISSUE THREAD

**Issue by spiffo-mom** [sharkster91/ExpandedHelicopterEvents#4]
[console.txt](https://github.com/sharkster91/ExpandedHelicopterEvents/files/6634874/console.txt)

**Comment by ChuckTheSheep** [comment]
Error: related to update() - I'm not sure what's causing this but it is most likely a strange mod compatibility issue.

`line 739: 	if distanceToTrueTarget <= (self.attackDistance*4) then`

```
STACK TRACE
-----------------------------------------
function: update -- file: ExpandedHelicopter01_Main.lua line # 739
function: updateAllHelicopters -- file: ExpandedHelicopter01_Main.lua line # 905

ERROR: General     , 1623366781721> ExceptionLogger.logException> Exception thrown java.lang.RuntimeException: __le not defined for operand at KahluaUtil.fail line:82.
ERROR: General     , 1623366781721> DebugLogStream.printException> Stack trace:
java.lang.RuntimeException: __le not defined for operand
	at se.krka.kahlua.vm.KahluaUtil.fail(KahluaUtil.java:82)
	at se.krka.kahlua.vm.KahluaThread.luaMainloop(KahluaThread.java:900)
	at se.krka.kahlua.vm.KahluaThread.call(KahluaThread.java:163)
	at se.krka.kahlua.vm.KahluaThread.pcall(KahluaThread.java:1980)
	at se.krka.kahlua.vm.KahluaThread.pcallvoid(KahluaThread.java:1812)
	at se.krka.kahlua.integration.LuaCaller.pcallvoid(LuaCaller.java:66)
	at se.krka.kahlua.integration.LuaCaller.protectedCallVoid(LuaCaller.java:139)
	at zombie.Lua.Event.trigger(Event.java:37)
	at zombie.Lua.LuaEventManager.triggerEvent(LuaEventManager.java:79)
	at zombie.gameStates.IngameState.updateInternal(IngameState.java:1601)
	at zombie.gameStates.IngameState.update(IngameState.java:1299)
	at zombie.gameStates.GameStateMachine.update(GameStateMachine.java:101)
	at zombie.GameWindow.logic(GameWindow.java:282)
	at zombie.core.profiling.AbstractPerformanceProfileProbe.invokeAndMeasure(AbstractPerformanceProfileProbe.java:71)
	at zombie.GameWindow.frameStep(GameWindow.java:721)
	at zombie.GameWindow.run_ez(GameWindow.java:637)
	at zombie.GameWindow.mainThread(GameWindow.java:464)
	at java.lang.Thread.run(Thread.java:745)
LOG  : General     , 1623366781724> -----------------------------------------
```

**Comment by ChuckTheSheep** [comment]
Note: This issue is still occuring even with update avoiding false-state/not yet launched helis.

Idea: if the issue isn't with launching it could be with unlaunching. Someone reported the issue occured with jets which are much faster.

**Comment by ChuckTheSheep** [comment]
Please confirm if this issue still persists. Thank you.

**Comment by ChuckTheSheep** [comment]
Fixed with 1d08dca96c52aaa39cc4e8e59bf7b35a9a67a05a and 903786da87b66e72f9ae1ea2bb2f31f59ad43fcf
